parity/reports/algod2indexer_full.yml

```diff
@@ -0,0 +1,150 @@
+Account:
+  properties:
+    closed-at-round:
+    - INDEXER: '{"description":"Round during w...'
+    - ALGOD: null
+    created-at-round:
+    - INDEXER: '{"description":"Round during w...'
+    - ALGOD: null
+    deleted:
+    - INDEXER: '{"description":"Whether or not...'
+    - ALGOD: null
+    sig-type:
+      description:
+      - INDEXER: '"Indicates what type of signature is used by this account, must be one of:\n* sig\n* msig\n* lsig\n* or null if unknown"'
+      - ALGOD: '"Indicates what type of signature is used by this account, must be one of:\n* sig\n* msig\n* lsig"'
```

Comment:
Looks ok?

Reply:
This is one of the problems with sharing these objects... this field is only set by the indexer. I'd say algod should be updated to match indexer, but also it doesn't really matter since it's not used.